plugins/builds/steps/logs.js

```diff
@@ -32,7 +33,13 @@ async function fetchLog({ baseUrl, linesFrom, authToken, page, sort }) {
 
     return new Promise((resolve, reject) => {
         requestStream
-            .on('error', e => reject(e))
+            .on('error', e => {
+                if (!e.message.includes(EOF_MESSAGE)) {
```

Comment:
look for explicit status code like this https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/builds/artifacts/get.js#L70
rather than string matching error message

Reply:
Yeah I originally tried that but there's no status code

Reply:
hmm, that sounds like it's a caught exception and not the original exception from `got` ?
This might be why https://github.com/screwdriver-cd/request/blob/master/index.js#L56 It's not the original got response. So https://github.com/screwdriver-cd/request/blob/master/index.js#L56 also needs to be fixed to use just 

https://github.com/screwdriver-cd/request/blob/master/index.js#L16